website/docs/api/plugins/plugin-google-gtag.md

```diff
@@ -67,7 +67,7 @@ Most Docusaurus users configure this plugin through the preset options.
 // Plugin Options: @docusaurus/plugin-google-gtag
 
 const config = {
-  trackingID: 'G-226F0LR9KE',
+  trackingID: 'G-226F0LR9XX',
```

Comment:
What is this new token? Maybe it belongs to someone? 😄

Reply:
Perhaps:

`G-999F0LR9XX`

Reply:
I've obfuscated a bit further still